geonode/base/models.py

```diff
@@ -1056,7 +1056,7 @@ def bbox(self):
         if self.bbox_polygon:
             bbox = self.bbox_polygon
             match = re.match(r'^(EPSG:)?(?P<srid>\d{4,6})$', self.srid)
-            srid = int(match.group('srid'))
+            srid = int(match.group('srid')) if match else 4326
```

Comment:
@afabiani Is this change in model.py intended in this PR? Does not look connected to the issue (visibility of user endpoint)

Reply:
Thanks @t-book nope this should not be included in this PR. Thanks for looking

Reply:
@t-book reverted